docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -276,6 +278,49 @@ content in a uniform way:
 
    Because it stands out better, use ``seealso`` for general references over the use of notes or adding links to the description.
 
+.. _semantic_markup:
+
+Semantic markup within module documentation
+-------------------------------------------
+
+You can use semantic markup to highlight option names, option values, and environment variables. The markup processor formats these highlighted terms in a uniform way. With semantic markup, we can modify how the output looks without changing underlying code.
+
+The correct formats for semantic markup are as follows:
+
+* ``O()`` for option names, whether mentioned alone or with values. For example: ``Required if O(state=present).`` and ``Use with O(force) to require secure access.``
+* ``V()`` for option values when mentioned alone. For example: ``Possible values include V(monospace) and V(pretty).``
+* ``RV()`` for return value names, whether mentioned alone or with values. For example: ``The module returns RV(changed=true) in case of changes.`` and ``Use the RV(stdout) return value for standard output.``
+* ``E()`` for environment variables. For example: ``If not set, the environment variable E(ACME_PASSWORD) will be used.``
```

Comment:
@felixfontein Is there no support for `E(name=value)` syntax?

Reply:
@mattclay no; the reason is (or was) that it maps directly to Sphinx's `:envvar:` role (https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-envvar), which doesn't support that either (or at least the documentation doesn't mention that). And simply nobody thought about it yet (or nobody bothered to mention that before you)...